### PR TITLE
Small Int EI

### PR DESCRIPTION
**Pull request by andrei-marinica**
Small int api, properly named and implemented.

Old functions kept for backwards compatibility.

Some more validation, tests.
The framework can disable the small int API, so there's a test with that too.

### CODE REVIEW

File: arwen/elrondapi/elrondei.go [hunk 1]
```diff
@@ -1359,9 +1384,13 @@ func int64storageLoad(context unsafe.Pointer, keyOffset int32, keyLength int32)
 
 	data := storage.GetStorage(key)
 
-	bigInt := big.NewInt(0).SetBytes(data)
+	valueBigInt := twos.SetBytes(big.NewInt(0), data)
```
Comment (sasurobert) [inline arwen/elrondapi/elrondei.go:1387]: is this backward compatible ?

Comment (camilbancioiu) [reply arwen/elrondapi/elrondei.go:1387]: Do we need the same in `int64storageStore()`?

Comment (andrei-marinica) [reply arwen/elrondapi/elrondei.go:1387]: no :(
will redesign

Comment (andrei-marinica) [reply arwen/elrondapi/elrondei.go:1387]: @camil yes, slipped me.
But will rewrite anyway

**Review by camilbancioiu — APPROVED** [review]

**Review by sasurobert — APPROVED** [review]

**Pull request merged**